plugins/labhub.py

```diff
@@ -1,7 +1,106 @@
-from errbot import BotPlugin
+import os
+import re
 
+import github3
 
-class Labhub(BotPlugin):
+from errbot import BotPlugin, re_botcmd
+
+
+class LabHub(BotPlugin):
     """
     GitHub and GitLab utilities
     """
+
+    INVITE_SUCCESS = {
+        'newcomers': 'Welcome @{}! :tada:\n\nTo get started, please '
+                     'follow our [newcomers guide](https://coala.io/newcomer).'
+                     'Most issues will be explained there and in linked pages -'
+                     ' it will save you a lot of time, just read it. *Really.*'
+                     "\n\n*Do not take an issue if you don't understand it on "
+                     'your own.* Especially if you are new you have to be '
+                     'aware that getting started with an open source community '
+                     'is not trivial: you will have to work hard and most '
+                     'likely become a better coder than you are now just as we '
+                     "all did.\n\nDon't get us wrong: we are *very* glad to "
+                     'have you with us on this journey into open source! We '
+                     'will also be there for you at all times to help you with '
+                     'actual problems. :)',
+        'developers': 'Wow @{}, you are a part of developers team now! :tada: '
```

Comment:
I guess we should have the same explanation for what entails being a developer as well.